### PR TITLE
Add readme for Frosty-CLI

### PR DESCRIPTION
**Pull request by DAPOTTIS**

### CODE REVIEW

**Comment by wannkunstbeikor** [comment]
Thanks can u add some disclaimer that mods made with v1 need to be converted first

**Comment by DAPOTTIS** [comment]
> Thanks can u add some disclaimer that mods made with v1 need to be converted first

done, added the oodle warning as well + included FrostyCLI in the main readme

File: README.md [hunk 1]
```diff
@@ -40,6 +40,10 @@ A GUI application which is used to create mods.
 ### FrostyModManager
 A GUI application which is used to select what mods to apply to the game.
 
+### FrostyCLI
+A CLI application which is used to create, update and apply mods. 
+See [Readme](https://github.com/DAPOTTIS/FrostyToolsuite/blob/Readme-Stuff/FrostyCli/README-CLI.md).
```
Comment (wannkunstbeikor) [inline README.md:45]: make sure this link is pointing to the correct repo, even if the file currently does not exist

**Comment by wannkunstbeikor** [comment]
can u rename the README-CLI to just README since its already in the cli directory

File: FrostyCli/README-CLI.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
+# Installation
+
+## Build from source
+
+### Install dependencies
+This project requires [.NET 7.0](https://learn.microsoft.com/en-us/dotnet/core/install/)
+
+### Build instructions
+Follow the [build instructions](https://github.com/DAPOTTIS/FrostyToolsuite?tab=readme-ov-file#from-source).
```
Comment (wannkunstbeikor) [inline FrostyCli/README-CLI.md:9]: same thing here with the link

File: FrostyCli/README-CLI.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+Example clip using the interactive mode to generate mod data:
+
+
+
+https://github.com/user-attachments/assets/3a3b93ea-e034-4ef6-b0fb-1729310cb111
+
+
+After generating a mod data folder, pass the datapath argument to the games launch options to apply the mods as such:
+
+```-datapath '<mod data path>'```
```
Comment (wannkunstbeikor) [inline FrostyCli/README-CLI.md:60]: also add that u can also use the environmental variable GAME_DATA_DIR instead

**Comment by wannkunstbeikor** [comment]
if those small issues are fixed im gonna merge it thanks a lot for ur work

**Comment by DAPOTTIS** [comment]
made those changes, lmk if there's anything else to do

**Comment by wannkunstbeikor** [comment]
yeah can u add the <path> thing which is after datapath also to gamedatadir

**Pull request merged**